README.md

```diff
@@ -156,6 +156,17 @@ const analyticsData = await api.analytics
     .getData();
 ```
 
+#### Get enrollments query
+
+```
+const analyticsData = await api.analytics
+    .getEnrollmentsQuery("Gs69Uw2Mom1", {
+        dimension: ["cYeuwXTCPkU", "fbfJHSPpUQD"],
+        enrollmentDate: "LAST_12_MONTHS,THIS_MONTH",
+    })
+    .getData();
```

Comment:
In the README, we typically include an example that would work with https://play.dhis2.org/dev/

Reply:
Non-related comment: also, in analytics.ts, add links to the section of the docs relevant. Example: https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/analytics.html#webapi_enrollment_analytics